bodhi/util.py

```diff
@@ -435,12 +435,10 @@ def request2html(context, request):
 def update2html(context, update):
     request = context.get('request')
 
-    if hasattr(update, 'title'):
-        title = update.title
-    else:
-        title = update['title']
```

Comment:
Huh, yeah.  :)  a01f5d3e236 was closer to right, yes.

We could consolidate it into one block like this:

```python
if isinstance(update, Update):
    title = update.title
    alias = update.alias
else:
    title = update['title']
    alias =
hasattr(update, 'title'):